kimfom01.Console.CodingTracker/CodingTrackerConsole/UserInput.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Globalization;
+
+namespace CodingTrackerConsole;
+
+public class UserInput
+{
+    private readonly Validation _dateTimeValidation = new();
+
+    public string GetDate()
+    {
+        Console.Write("Enter date (mm-dd-yyyy): ");
+        var date = Console.ReadLine();
+
+        while (!_dateTimeValidation.IsValidDate(date))
+        {
+            Console.WriteLine("Invalid date! Expected format (mm-dd-yyyy)");
+            date = Console.ReadLine();
+        }
+
+        return date;
+    }
+
+    public string GetTime()
+    {
+        Console.Write("Enter time (hh:mm): ");
+        var time = Console.ReadLine();
+
+        while (!_dateTimeValidation.IsValidTime(time))
+        {
+            Console.Write("Invalid time! Expected format (hh:mm): ");
+            time = Console.ReadLine();
+        }
+
```

Comment:
At the moment I can input the end date before the start date! That shouldn't be allowed 😟
![image](https://user-images.githubusercontent.com/108711476/211082574-b25f1fa8-2971-4e5c-a87e-4365dde47b01.png)